rails/rails_trunk/rails_benchmarks/Dockerfile

```diff
@@ -6,7 +6,7 @@ RUN apt-get install -y libncurses5-dev libmysqlclient-dev sqlite3 libsqlite3-dev
 
 RUN git clone --verbose --branch master --single-branch https://github.com/ruby-bench/ruby-bench-suite.git
 RUN git clone --verbose --branch master --single-branch https://github.com/rails/rails.git
-RUN /bin/bash -l -c "gem install bundler --no-ri --no-rdoc && cd /rails && bundle install --without test doc job"
+RUN /bin/bash -l -c "gem install bundler --no-ri --no-rdoc && cd /rails && export LANG=C.UTF-8 && bundle install --without test doc job"
```

Comment:
This is not the preferred way to declare environment variables in a Dockerfile.

see https://docs.docker.com/engine/reference/builder/#/env